variables.tf

```diff
@@ -351,7 +351,20 @@ variable "create_security_group" {
 
 variable "port" {
   description = "The port on which the DB accepts connections"
+  type        = number
+  default     = 3306
```

Comment:
Should we set 3306 as the default? Given the multiple db engines..

Reply:
The default is 3306, in cases where a different port number is required we provide the port number